### ISSUE TITLE
New BRICS members Sentiment Analysis

### ISSUE THREAD

**Issue by abhisheks008** [abhisheks008/ML-Crate#518]
### ML-Crate Repository (Proposing new issue)
:red_circle: **Project Title** : New BRICS members Sentiment Analysis
:red_circle: **Aim** : The aim of this project is to analyze the sentiments of the new members  of BRICS.
:red_circle: **Dataset** : https://www.kaggle.com/datasets/syedali110/6-new-brics-members-sentiment-analysis
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Participant ID (If not, then put NA) :
- Approach for this Project :
- What is your participant role? (Mention the Open Source Program name. Eg. HRSoC, GSSoC, GSOC etc.)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by Yuvika-14** [comment]
Name :Yuvika Singh
git hub profile link:https://github.com/Yuvika-14/Yuvikademo.git
approch: ensemble methods, gradient boosting, neural networks , xgboost. 
Take care of the missing data if there are any categorical values then will use one hot encoder or label encoder depending upon the issue.

**Comment by abhisheks008** [comment]
One issue at a time @Yuvika-14

**Comment by abhisheks008** [comment]
Issue assigned to you @Yuvika-14

**Comment by abhisheks008** [comment]
Hi @Yuvika-14  as this a machine learning project with proper implementation, I know it requires time to put forward the highest quality in the project. You can take as much time as you, but make sure you are creating something which should be of highest quality.

**Comment by tanuj437** [comment]
Full name : Tanuj Saxena
GitHub Profile Link : https://github.com/tanuj437
Participant ID (If not, then put NA) : NA
Approach for this Project : Best approach can be done with Bagging and Boosting approach along with that can try for neural network as if activation function provide better result after doing EDA approaches properly
Participant of SSOC

**Comment by abhisheks008** [comment]
Implement 5-6 models for this dataset.

Assigned @tanuj437
